Mycelium/scripts/frontend-react/src/components/HexGridSVG.jsx

```
@@ -23,6 +23,7 @@ const HexGridSVG = ({
   if (!pattern || pattern.length === 0) return null;
 
   const hexSize = 20; // Distance from center to vertex
+  const clipPathId = useId(); // Generate unique ID for this component instance
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

Hooks should not be called after an early return.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)